### ISSUE TITLE
Deploying Django App.. 

### ISSUE THREAD

**Issue by petey284** [dc-wdi-22/seekr-frontend#8]
Can you send instructions on how to deploy a django app?

**Comment by aspittel** [comment]
Yep! I like this: https://simpleisbetterthancomplex.com/tutorial/2016/08/09/how-to-deploy-django-applications-on-heroku.html#disqus_thread for django heroku deployment just make sure to use an environmental variable or that decouple library for your SECRET_KEY as well!

**Comment by petey284** [comment]
We are a little confused about how to make it work? The example project is configured differently than our django app: https://github.com/vitorfs/bootcamp/

We are stuck on the first step.

**Comment by aspittel** [comment]
Actually found something else I'm testing real quick
